questionnaire/2020/questions.yaml

```diff
@@ -215,6 +215,20 @@
     - A little
     - A lot
 
+  - title: What connection methods / transports are being utilized as part of your network automation implementation?
+    type: Multiple choice 
+    id: transport-protocol
+    responses:
+    - CLI
+    - Netconf
+    - Restconf
+    - gNMI
+    - gRPC
+    - HTTP/HTTPS
+    - Console
+    - Not sure
```

Comment:
Sadly, I think we should add SNMP to the list. I know we mention it elsewhere under "data sources are you using to detect problems in your network" but I think it belongs here also.

Reply:
Do automation tools utilize SNMP or just for monitoring?

Reply:
I know of at least one security automation tool that uses SNMP to change the VLAN membership on access switch ports... Maybe it's a one-off. If you guys are against listing it, I won't lose sleep over it. But I bet there is some SMNPset-based  automation out there...

Reply:
I'd be ok to have it in, I just know SNMP isn't a pervasive means for configuration automation but maybe for monitoring / logging

Reply:
I'm honestly indifferent on this - I'll leave this to yall

Reply:
I think it would be confusing, I know some people might be using it to change the config but I would prefer to not have it here